app/brave_settings_strings.grdp

```diff
@@ -1473,6 +1473,9 @@
   <message name="IDS_SETTINGS_CLOSING_LAST_TAB_OPTION_LABEL" desc="The text for settings option">
     Close window when closing last tab
   </message>
+  <message name="IDS_SETTINGS_SHOW_FULLSCREEN_REMINDER_OPTION_LABEL" desc="The text for fullscreen reminder settings option">
+    Show full screen reminder to press Esc on exit
```

Comment:
Hmm, when I enter fullscreen mode on Windows the message says "To exit full screen, press F11". It might be better to reword this as "Show exit instructions when entering full screen mode".

Reply:
Hi @emerick 

We actually intend to hide only the `Show fullscreen reminder to press Esc to exit`, which pops up only when the video goes into fullscreen mode (basically tab-initiated fullscreen). Hence, we will show `To exit fullscreen, press F11` regardless of this preference.

FYR: https://github.com/brave/brave-browser/issues/38928#issuecomment-2166653019

I think renaming the PR title and description will avoid this confusion.

Reply:
I see, I didn't realize that was a separate mode.